code/modules/cargo/packs.dm

```diff
@@ -1306,32 +1347,68 @@
 /datum/supply_pack/medical/synthflesh
 	name = "Synthflesh resupply pack"
 	desc = "Contains four 100u cartons of synthflesh in case the cloner ran out of it."
-	cost = 3000
+	cost = 1400
 	contains = list(/obj/item/reagent_containers/food/drinks/bottle/synthflesh,
 					/obj/item/reagent_containers/food/drinks/bottle/synthflesh,
 					/obj/item/reagent_containers/food/drinks/bottle/synthflesh,
 					/obj/item/reagent_containers/food/drinks/bottle/synthflesh)
 	crate_name = "rusty freezer"
 	crate_type = /obj/structure/closet/crate/freezer
 
-/datum/supply_pack/medical/firstaidbruises_single
+datum/supply_pack/medical/bruisekits
+	name = "Basic Treatment Kits Crate"
+	desc = "Contains three basic aid kits focused on basic types of damage in a simple way."
+	cost = 1000
+	small_item = TRUE
+	contains = list(/obj/item/storage/firstaid/regular,
+					/obj/item/storage/firstaid/regular,
+					/obj/item/storage/firstaid/regular)
+	crate_name = "basic wound treatment kits crate"
+
+datum/supply_pack/medical/bruisekits
+	name = "Bruise Treatment Kits Crate"
+	desc = "Contains three first aid kits focused on healing bruises and broken bones."
+	cost = 1000
+	small_item = TRUE
+	contains = list(/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/brute)
+	crate_name = "brute treatment kits crate"
+
+/datum/supply_pack/medical/burnkits
 	name = "Bruise Treatment Kit Single-Pack"
```

Comment:
Fix this

Reply:
You mean how the basic kits pack was supply_pack/medical/brutekits?
I saw that bug and fixed it unless there is another you mean. I don't see one here.